catroid/src/main/java/org/catrobat/catroid/content/bricks/SetLookBrick.java

```diff
@@ -231,7 +231,7 @@ public Brick clone() {
 
 	@Override
 	public List<SequenceAction> addActionToSequence(Sprite sprite, SequenceAction sequence) {
-		sequence.addAction(sprite.getActionFactory().createSetLookAction(getSprite(), look, wait));
+		sequence.addAction(sprite.getActionFactory().createSetLookAction(sprite, look, wait));
```

Comment:
@michael-fischl 
could you check if the [SetBackgroundBrick](https://github.com/michael-fischl/Catroid/blob/852e9cc42ffeb64fcb4f63f5ef0c078a17846db6/catroid/src/main/java/org/catrobat/catroid/content/bricks/SetBackgroundBrick.java) and [SetBackgroundAndWaitBrick](https://github.com/michael-fischl/Catroid/blob/852e9cc42ffeb64fcb4f63f5ef0c078a17846db6/catroid/src/main/java/org/catrobat/catroid/content/bricks/SetBackgroundAndWaitBrick.java) are still working? Thank you!
See PR #1874 and CAT-1968